src/client/app/components/Toggler/Toggler.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import styles from './Toggler.less'
+import TogglerItem from './TogglerItem'
+
+export default class Toggler extends React.Component {
+  constructor (props) {
```

Comment:
There is no reason to define `constructor` just for passing props to a base class.